packages/react-components/react-portal/src/components/Portal/usePortal.test.ts

```diff
@@ -4,28 +4,26 @@ describe('toMountNodeProps', () => {
   it('handles HTMLElement', () => {
     const element = document.createElement('div');
 
-    expect(toMountNodeProps(element)).toMatchObject({
-      element,
-    });
+    expect(toMountNodeProps(element)).toEqual({ element });
```

Comment:
starting with jest/ts-jest 29 looks like `toMatchObject` has some issues when using with DOM

<img width="614" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/481fadc9-d2de-444e-a593-062c1746589d">

I found only following one which was fixed:
https://github.com/jestjs/jest/issues/6730

I refactored the code to more appropriate matcher and which also better exposed wrong test for `undefined`